src/execution_plan/ops/shared/create_functions.c

```diff
@@ -171,8 +140,8 @@ PendingCreations NewPendingCreationsContainer(NodeCreateCtx *nodes, EdgeCreateCt
 	pending.node_labels = array_new(int *, 0);
 	pending.created_nodes = array_new(Node *, 0);
 	pending.created_edges = array_new(Edge *, 0);
-	pending.node_properties = array_new(PendingProperties *, 0);
-	pending.edge_properties = array_new(PendingProperties *, 0);
+	pending.node_properties = array_new(Entity, 0);
```

Comment:
Seems like `pending.node_attributes = array_new(AttributeSet, 0);` is more appropriate